eng/.emscripten

```diff
@@ -4,7 +4,7 @@ LLVM_ROOT = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_LLVM_ROOT', ''))
 NODE_JS = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_NODE_JS', ''))
 BINARYEN_ROOT = os.path.expanduser(os.getenv('DOTNET_EMSCRIPTEN_BINARYEN_ROOT', ''))
 
-FROZEN_CACHE = True
+FROZEN_CACHE = os.path.expanduser(os.getenv('FROZEN_CACHE', 'True'))
```

Comment:
`os.path.expanduser` doesn't make sense here since FROZEN_CACHE doesn't contain a file path but a boolean:

```suggestion
FROZEN_CACHE = bool(os.getenv('FROZEN_CACHE', 'True'))
```